iterations_runners/iterations_runner.py

```diff
@@ -70,18 +72,29 @@
 krun_get_aperf = libkruntime.krun_get_aperf
 krun_get_mperf = libkruntime.krun_get_mperf
 
+def usage():
+    print(__doc__)
+    sys.exit(1)
+
 # main
 if __name__ == "__main__":
-    if len(sys.argv) != 6:
-        sys.stderr.write("usage: iterations_runner.py <benchmark> "
-                         "<# of iterations> <benchmark param> <debug flag> "
-                         "<instrument flag>\n")
-        sys.exit(1)
+    num_args = len(sys.argv)
+    if num_args < 6:
+        usage()
 
-    benchmark, iters, param, debug, instrument = sys.argv[1:]
+    benchmark, iters, param, debug, instrument = sys.argv[1:6]
     iters, param, debug, instrument = \
         int(iters), int(param), int(debug) == 1, int(instrument) == 1
 
+    if instrument:
+        # The new instrumentation switches are unused here, as the PyPy
+        # instrumentation code uses the legacy instrumentation framework. We
+        # still check the right number of args is passed.
+        if num_args != 8:
```

Comment:
Maybe make 8 and 6 variables here?

Reply:
Ah, it's hardly worth it in this case. Do you feel strongly?